fsm/leave_parent_ns_synced.go

```diff
@@ -197,3 +197,27 @@ func LeaveParentNsSyncedAction(z *music.Zone) bool {
 
 	return true
 }
+
+func LeaveParentNsSyncedConfirmCsyncRemoval(zone *music.Zone) bool {
```

Comment:
Just no.

You've heard the story of the guy that invented a filesystem with infinite compression? Any file could be compressed to a single byte. Only problem was that it required the filesystem to support arbitrarily long filenames...

Reply:
True story. I am not happy with that one myself.  The problem is that we have some naming issues throughout MUSIC. I would be more that happy to address these and take the time to decide on how we want it to be.

Reply:
You know my view: trying to document via long variable names is not the way to go. In this case we're talking about a function that is called from perhaps two places in total. Then a better solution is comments in the code.

If an identifier name is used in a gazillion places (like the discussion about "z" vs "zone") then it is more important to find the "right" name of the identifier.